index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <title>Dr. Norman Borlaug</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+    <body>
+      <main id=""main>
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <div id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+            <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>

```

Comment:
This string seems too long, you can split it to 2-3 lines for better readability.